#! /usr/bin/env python

PACKAGE = 'sound_classification'

from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()

gen.add("window_size", double_t, 0, "Window size of sound input", 1.0, 0.0, 10.0)

exit(gen.generate(PACKAGE, PACKAGE, "AudioTagging"))
